browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -3522,7 +3522,7 @@ index 7a790bb23c8f37af57e4284b367cad9d679a434f..a8b090780ff0811a0ac7614ee6ad3da1
 +        absoluteBounds.setHeight(LayoutUnit(std::max(*height, 1.0)));
 +    }
 +    ScrollAlignment alignment = ScrollAlignment::alignCenterIfNeeded;
-+    alignment.m_disableMinThreshold = true; // Disable RenderLayer minium horizontal scroll threshold.
++    alignment.m_enableLegacyHorizontalVisibilityThreshold = true;
```

Comment:
Overlooked this, I assume the value assigned to the new variable should be `false`

Reply:
Fixed in https://github.com/microsoft/playwright/pull/11722